JSDemos/Demos/DataGrid/CustomNewRecordPosition/description.md

```diff
@@ -0,0 +1,25 @@
+The DataGrid API includes the [newRowPosition](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#newRowPosition) property that allows you to set one of the following predefined new record positions:
+
+- *"viewportTop"* (default)           
+Insert a new row at the top of the viewport.
+
+- *"viewportBottom"*          
+Insert a new row at the bottom of the viewport.
+
+- *"pageTop"*         
+Insert a new row at the top of the current page. In virtual and infinite [scrolling modes](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/scrolling/#mode), *"pageTop"* works like *"viewportTop"*. 
```

Comment:
"pageTop"* works like *"viewportTop"*. Do you mean the "pageTop"* position matches  *"viewportTop"*?